modules/server/src/main/ts/bedrock/auto/DriverLoader.ts

```diff
@@ -31,64 +21,34 @@ const browserExecutables: Record<string, string[]> = {
   'MicrosoftEdge': [ 'msedgedriver', 'MicrosoftWebDriver' ]
 };
 
-const execLoader = (exec: string, driverArgs: string[] = []) => {
-  const api = {} as DriverAPI;
-  api.start = (args = []) => {
-    const finalArgs = driverArgs.concat(args);
-    api.defaultInstance = crossSpawn(exec, finalArgs);
-    return api.defaultInstance;
-  };
-
-  api.stop = () => {
-    if (api.defaultInstance) {
-      api.defaultInstance.kill();
-      api.defaultInstance = null;
-    }
-  };
-
-  return api;
-};
-
-const findNpmPackage = (driverDeps: string[]) => Arr.findMap(driverDeps, (driverDep) => {
-  try {
-    return require(driverDep);
-  } catch (e) {
-    return null;
-  }
-});
-
-const findExecutable = (execNames: string[]) => Arr.findMap(execNames, (execName) => {
-  return which.sync(execName, { nothrow: true });
-});
-
 const loadPhantomJs = (settings: DriverSettings) => {
-  const api = execLoader('phantomjs', [ '--remote-debugger-port=' + settings.debuggingPort ]);
+  const api = ExecUtils.execLoader('phantomjs', [ '--remote-debugger-port=' + settings.debuggingPort ]);
 
   // Patch the start function to remap the arguments
   const origStart = api.start;
   api.start = (args = []) => {
-    const patchedArgs = args.map((arg) => {
+    const patchedArgs = args.map((arg: any) => {
```

Comment:
I wonder if we should circle back on this one and fix the types. It seems that all invocations of this function is either without arguments or with a first argument with an array of strings. I'm always a bit uneasy when the `any` word goes in but this seems to have been like this for a long time.

Reply:
These particular types are tricky because of the driver differences between loading via npm and having executable drivers and typescript not behaving nicely with rest parameters. At least at this point we can assume `string`s but typing the rest parameters did not go as expected, unfortunately